docs/src/modules/components/CarbonAds.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
What's the pure for?

Reply:
Leftover from the original Carbon implementation.